sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/_input_output_helpers.py

```diff
@@ -325,6 +334,7 @@ def to_rest_data_outputs(outputs: Dict[str, Output]) -> Dict[str, RestJobOutput]
                     asset_version=output_value.version,
                     uri=output_value.path,
                     mode=OUTPUT_MOUNT_MAPPING_TO_REST[output_value.mode.lower()] if output_value.mode else None,
+                    pathOnCompute=output_value.path_on_compute if output_value.path_on_compute else None,
```

Comment:
literal should not have such property?

Reply:
yes ,this will be deleted.

Reply:
for output we donnot have literal output here, so all this outputs could support path_on_compute
![image](https://github.com/Azure/azure-sdk-for-python/assets/110205279/46ca4644-34d5-4f02-a8f0-854269dad74d)

![image](https://github.com/Azure/azure-sdk-for-python/assets/110205279/bcce1393-c74f-4043-8658-63a9d155a6cc)